ui-admin/src/forms/FormContentEditor.tsx

```diff
@@ -62,15 +62,14 @@ export const FormContentEditor = (props: FormContentEditorProps) => {
     <div className="FormContentEditor d-flex flex-column flex-grow-1">
       <Tabs
         activeKey={activeTab ?? undefined}
-        className="mb-1"
         mountOnEnter
         unmountOnExit
         onSelect={setActiveTab}
       >
         {userHasPermission(user, portal.id, 'prototype_tester') &&< Tab
           disabled={activeTab !== 'designer' && !tabsEnabled}
           eventKey="designer"
-          title="Designer"
+          title={<>Designer<span className='badge bg-primary fw-light ms-2'>LEGACY</span></>}
```

Comment:
maybe move this tab all the way over to the right so it's less likely we'll get confused and click the wrong designer